ovn_cluster.sh

```diff
@@ -18,10 +18,14 @@ OS_IMAGE=${OS_IMAGE:-"quay.io/fedora/fedora:latest"}
 OS_IMAGE_PULL_RETRIES=${OS_IMAGE_PULL_RETRIES:-40}
 OS_IMAGE_PULL_INTERVAL=${OS_IMAGE_PULL_INTERVAL:-5}
 USE_OVSDB_ETCD=${USE_OVSDB_ETCD:-no}
-CENTRAL_NAME="ovn-central"
 CHASSIS_PREFIX="${CHASSIS_PREFIX:-ovn-chassis-}"
 GW_PREFIX="ovn-gw-"
 
+CENTRAL_COUNT=${CENTRAL_COUNT:-1}
+CENTRAL_PREFIX="ovn-central-az"
+CENTRAL_NAME="ovn-central-az1"
```

Comment:
To allow users (e.g., ovn-heater) to better control how AZs are created it would be useful if these were configurable, e.g.:
```
CENTRAL_PREFIX="${CENTRAL_PREFIX:ovn-central-az}"
CENTRAL_NAME="${CENTRAL_PREFIX}-1"
```

Reply:
@dceara ack, I will fix it